.github/workflows/build-oci.yaml

```diff
@@ -67,7 +66,28 @@ jobs:
         with:
           name: mapt-${{ env.ARCH_TYPE }}
           path: mapt*
+  combine-artifacts:
+    name: combine-artifacts
```

Comment:
May you can add a condition to combine only on pull_request....as for push you directly push to quay